local/client.go

```diff
@@ -102,6 +120,8 @@ func NewClient(opts ...OptionFunc) *Client {
 			MulticastAddressUDP4: DiscoveryAddressUDP4,
 			MulticastAddressUDP6: DiscoveryAddressUDP6,
 		},
+		enableDTLS:   true,
+		enableTCPTLS: true,
```

Comment:
You could name these flags `disable` instead of `enable` to rely on the zero value as the default. Moreover, it would be nice to decouple the connection configuration for reuse by the OTM client and possibly other components.